From 033cdfbd42329fcd008b389f0482cc7075ea5cf2 Mon Sep 17 00:00:00 2001
From: Daniel Schultz <d.schultz@phytec.de>
Date: Tue, 19 Dec 2017 16:47:05 +0100
Subject: [PATCH] srvkm: common: Add fallthrough attribute to case

The new GCC versions (7.X) will forbid fallthroughs in switch case
statements. Add a attribute to allow this exception.

Signed-off-by: Daniel Schultz <d.schultz@phytec.de>
---
 eurasia_km/services4/srvkm/common/devicemem.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/eurasia_km/services4/srvkm/common/devicemem.c b/eurasia_km/services4/srvkm/common/devicemem.c
index baf7c41..d3f0db9 100644
--- a/eurasia_km/services4/srvkm/common/devicemem.c
+++ b/eurasia_km/services4/srvkm/common/devicemem.c
@@ -1071,6 +1071,7 @@ PVRSRV_ERROR FreeMemCallBackCommon(PVRSRV_KERNEL_MEM_INFO *psMemInfo,
 			case PVRSRV_MEMTYPE_ION:
 			case PVRSRV_MEMTYPE_DMABUF:
 				freeExternal(psMemInfo);
+				 __attribute__((fallthrough));
 			case PVRSRV_MEMTYPE_DEVICE:
 			case PVRSRV_MEMTYPE_DEVICECLASS:
 #if defined(SUPPORT_ION)
-- 
2.7.4

